templates/email/default/archive-old-enquiries.html

```diff
@@ -14,7 +14,7 @@ <h1 style="[% h1_style %]">Your reports on [% site_name %]</h1>
     Hello [% user.name %],
```

Comment:
This is generalised, thanks, but I'd hoped from my previous comment that e.g. the Hounslow/Westminster ones would be removed and any slight differences they had (if any) would be done as included sub-templates. But no matter for now.